src/pages/workspaces/List.js

```diff
@@ -251,19 +253,47 @@ export const WorkspaceList = () => {
             ])
           },
           size: { basis: 120, grow: 1, shrink: 0 }
+        }, {
+          headerRenderer: () => div({ className: 'sr-only' }, ['Last Workflow Submitted Status']),
+          cellRenderer: ({ rowIndex }) => {
+            const { ...workspace } = sortedWorkspaces[rowIndex]
+            const lastRunStatus = workspaceSubmissionStatus(workspace)
+
+            return div({ style: { ...styles.tableCellContainer, paddingRight: 0 } }, [
+              div({ style: styles.tableCellContent }, [
+                h(WorkspaceSubmissionStatusIcon, {
+                  status: lastRunStatus,
+                  loadingSubmissionStats
+                })
+              ])
+            ])
+          },
+          size: { basis: 30, grow: 0, shrink: 0 }
+        }, {
+          headerRenderer: () => div({ className: 'sr-only' }, ['Cloud Platform']),
+          cellRenderer: ({ rowIndex }) => {
+            const { workspace: { cloudPlatform } } = sortedWorkspaces[rowIndex]
+            return div({ style: { ...styles.tableCellContainer, paddingRight: 0 } }, [
+              div({ style: styles.tableCellContent }, [
+                Utils.switchCase(cloudPlatform,
+                  ['Gcp', () => h(CloudGcpLogo, { title: 'Google Cloud Platform', role: 'img' })],
+                  ['Azure', () => h(CloudAzureLogo, { title: 'Microsoft Azure', role: 'img' })])
+              ])
+            ])
+          },
+          size: { basis: 30, grow: 0, shrink: 0 }
         }, {
           headerRenderer: () => div({ className: 'sr-only' }, ['Actions']),
           cellRenderer: ({ rowIndex }) => {
-            const { accessLevel, workspace: { workspaceId, namespace, name }, ...workspace } = sortedWorkspaces[rowIndex]
+            const { accessLevel, workspace: { workspaceId, namespace, name } } = sortedWorkspaces[rowIndex]
             if (!Utils.canRead(accessLevel)) {
-              // No menu shown if user does not have read acccess.
-              return null
+              // No menu shown if user does not have read access.
+              return div({ className: 'sr-only' }, ['Action menu is not available due to access level.'])
```

Comment:
Do we need this message? For non-screen reader users, we're ok with no content here.

Reply:
If you don't have it, you get "Blank". Plus it renders in an unattractive way. See https://github.com/DataBiosphere/terra-ui/pull/3303#discussion_r942526064

Reply:
Ah, ok.

This could possibly be reworded to be more similar to other permission messages.

```suggestion
              return div({ className: 'sr-only' }, ['You do not have permission to perform actions on this workspace.'])
```

https://github.com/DataBiosphere/terra-ui/blob/dev/src/libs/utils.js#L189-L202